src/libYARP_OS/src/AuthHMAC.cpp

```diff
@@ -35,6 +35,7 @@ using namespace yarp::os;
 AuthHMAC::AuthHMAC() :
         authentication_enabled(false)
 {
+    memset(&context, 0, sizeof(HMAC_CONTEXT));
```

Comment:
:+1: done here https://github.com/robotology/yarp/pull/1463/commits/4651d0271e8d32911787164b55766f6c7eb56a85 but I think coverity will keep complaining because of the conditional returning.